pkg/dunner/dunner.go

```diff
@@ -51,6 +51,10 @@ func Do(_ *cobra.Command, args []string) {
 func ExecTask(configs *config.Configs, taskName string, args []string) {
 	var async = viper.GetBool("Async")
 	var wg sync.WaitGroup
+
+	if _, exists := configs.Tasks[taskName]; !exists {
```

Comment:
@ayushjn20 Please address this

Reply:
taskName is given by the CLI and not the dunner file. I believe validation is done for the dunner file. Could you please explain with file and line number, where should the check be implemented? @apoorvam

Reply:
> Also, this change is included in previous PR too.
Correction, changes are present in the *next* PR too. 
Those are the same commits, I could not proceed without resolving go fmt errors so already rebased it.

Reply:
Right that taskName comes from CLI. We cannot add it to validation/config package itself. 

We can add [Validate call](https://github.com/leopardslab/dunner/blob/4e9b9bbfba09193200386301e970bb6ecdfb0615/pkg/dunner/dunner.go#L38) and this check in a function which does all validation tasks instead of mixing up with execution and validation. That is optional though, as long as you can cover this code with unit tests.

Reply:
@apoorvam Can you then please approve the PR?

Reply:
@agentmilindu I don't yet see the unit test added. However, code looks fine.